lib/app.js

```diff
@@ -205,11 +204,85 @@ module.exports = function(tilelive, options) {
             headers["content-encoding"] = headers["content-encoding"] || "gzip";
           }
 
-          res.set(populateHeaders(headers, params, { 200: true }));
-          return res.send(data);
+          return callback(null, data, populateHeaders(headers, params, { 200: true }));
         });
       });
     });
+  };
+
+  app.get(tilePattern, function(req, res, next) {
+    var z = req.params.z | 0,
+        x = req.params.x | 0,
+        y = req.params.y | 0,
+        scale = getScale(req.params.scale),
+        format = req.params.format;
+    return getTile(z, x, y, scale, format, function(err, data, headers) {
+        if (err && !err.status) {
+          return next(err);
+        }
+        res.set(headers);
+        res.status((err && err.status) || 200);
+        return res.send((err && err.message) || data);
+    }, res, next);
+  });
+
+  var processStaticMap = function(areaParams, req, res, next) {
+    var scale = getScale(req.params.scale),
+        format = req.params.format,
+        params = {
+          zoom: req.params.z | 0,
+          scale: scale,
+          bbox: areaParams.bbox,
+          center: areaParams.center,
+          format: format,
+          getTile: function(z, x, y, callback) {
+            return getTile(z, x, y, scale, format, function(err, data, headers) {
+              if (!err && data == null) {
```

Comment:
You probably want to keep `data` as null, otherwise abaculus will fail to stitch maps that are partially transparent (due to tiles that aren't present):

https://github.com/mapbox/abaculus/blob/master/index.js#L177
https://github.com/mapbox/abaculus/blob/master/index.js#L159